README.md

```diff
@@ -22,7 +22,7 @@ The `setup_for_wrf.py` script does the following:
 * Performs substitutions of the config file. For example, if used, the shell environment variable `${HOME}` will be replaced by its value when interpreting the script. The variable `wps_dir` is defined within the config file, and if the token `${wps_dir}` appears within the configuration entries, such tokens will be replaced by the value of this variable.
 * Configure the main coordination script
 * Loop over the required WRF jobs, performing the following:
-  * Check if the WRF input files for this run are available (`wrfinput_d0?`). If not, perform the following:
+  * Check if the WRF input files for this run are available (`wrfinput_d01`). If not, perform the following:
```

Comment:
The scripts checks for the file `wrfinput_d01`. Not sure if that's a typo or if there can be more files `wrfinput_d02`, `wrfinput_d03` and so on.

Reply:
There can be more than one domain. Its not currently done, but we can "nest" domains inside each other